### ISSUE TITLE
There shouldn't be "Duplicate name..." warnings with pedantic flag set to true

### ISSUE THREAD

**Issue by jpchristen** [bndtools/bnd#2803]
This message spams the warnings as soon as you set `-pedantic: true`. It would be nice if you don't have to use `-fixupmessages Duplicate name` in order to get rid of those warnings as those duplicates seem to be common.

**Comment by bjhargrave** [comment]
Well the whole point of `-pedantic` is to be very pedantic. :-) As you see, with `-fixupmessages` you can elide the pedantic warnings which are of no interest to you. 

To me this is sort of like FindBugs where you can elide the FindBugs warnings you are not concerned about. But you don't ask FindBugs to not report the warning to others who may care about that warning.

**Comment by pkriens** [comment]
Paul is right, this particular message is actually wrong since duplicates are valid

**Comment by bjhargrave** [comment]
It is not that they are invalid. After all this is a pedantic warning. But it generally probably not what you expect. Like exporting a package multiple times is a rare thing but still valid. So the pedantic warning can advise you to check your work and then elide the warning (like SuppressWarnings) once you are satisfied you are doing what you intended.

**Comment by pkriens** [comment]
Yes, but we should look at the actual header. I think this warning is on the Provide/Require capability and there the key is the namespace. Very hard not to duplicate :-(

**Comment by bjhargrave** [comment]
It is `OSGiHeader` which is used to parse headers for `Parameters` so it is used a lot and all over the place.

https://github.com/bndtools/bnd/blob/9b431f5d68be2fcbff957b53f5b82b9f4944252e/biz.aQute.bndlib/src/aQute/bnd/header/OSGiHeader.java#L92-L100

**Comment by pkriens** [comment]
That is too wide. When I wrote that I thought it was going to be very rare but with the Require/Capability we made it extremely common. So that code probably needs to go and then add a check in Verify?

**Comment by bjhargrave** [comment]
OK, you may also want to move the check on duplicate attribute/directive names as well.

https://github.com/bndtools/bnd/blob/9b431f5d68be2fcbff957b53f5b82b9f4944252e/biz.aQute.bndlib/src/aQute/bnd/header/OSGiHeader.java#L70-L79
